disnake/ext/commands/bot.py

```diff
@@ -85,6 +92,9 @@ class Bot(BotBase, InteractionBotBase, disnake.Client):
             Changes the log level of corresponding messages from ``DEBUG`` to ``INFO`` or ``print``\\s them,
             instead of controlling whether they are enabled at all.
 
+        .. deprecated:: 2.7
+            Replaced with ``command_sync``.
+
```

Comment:
Missing the `command_sync` documentation

Reply:
https://github.com/DisnakeDev/disnake/pull/806/commits/9a6d7751fb8092a3ae752662ba678e7300fb3835